composer.json

```diff
@@ -64,6 +80,10 @@
         "drupal/stage_file_proxy": "^2.0",
         "drupal/structure_sync": "^2.0",
         "drupal/ultimate_cron": "^2.0@alpha",
+        "ecc_cludo_search/ecc_cludo_search": "^1.0",
```

Comment:
Change these to match content ownership e.g.
`essexcountycouncil/ecc_cludo_search`
This means changing composer.json in the module repos, and also updating Intranet.

Reply:
The vendor has been changed in the module repos so composer.json should be updated here (already done for Intranet)